contrib/yggdrasil-brute-simple/.gitignore

```diff
@@ -0,0 +1,7 @@
+*
```

Comment:
I wonder if you intend this directory to be a submodule instead? https://git-scm.com/book/en/v2/Git-Tools-Submodules